resources/views/service/index_registration.blade.php

```diff
@@ -26,22 +26,21 @@
         </div>
         <div class="row">
             <div class="col">
-                <h2>Results:</h2>
                 <table>
                     <thead>
                     <tr>
                         <td>Name</td>
-                        <td>CC ID</td>
                         <td>Voucher Entitlement</td>
+                        <td>RV-ID</td>
```

Comment:
Is RV-ID human?

Reply:
Was Kat's suggestion as she thought it would be more user friendly, might remove the dash though

Reply:
Cool. we've time to change it during testing.

Reply:
Yep! Have left as is.